packages/core/src/lib/NftCard/types.ts

```diff
@@ -38,4 +38,14 @@ export interface INftCardProps {
      * custom details component
      */
     customDetails: ReactNode;
+
+    /**
+     * set card background color
+     */
+    backgroundColor: string;
+
+    /**
+     * set border for details section
+     */
+    detailsBorder: string | 'none';
```

Comment:
this seems strange. maybe it should be optional? but the dev cant even use it to set a border... If its an on / off switch use a boolean like hasBorderStyles. But best DX is to offer a `border?: string` so the dev can set `border: 1px solid color` and if its left unset it shows what you have now... or you set current styles as the default value on line 17 of `packages/core/src/lib/NftCard/NftCard.tsx`

Reply:
- Yes this is optional.
- It's not named as `border` because this border is for the component inside card.
- will remove extra `none` ✅